letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -768,6 +768,14 @@ elif [ -f /etc/redhat-release ]; then
     }
     USE_PYTHON_3=1
     BOOTSTRAP_VERSION="BootstrapRpmPython3 $BOOTSTRAP_RPM_PYTHON3_VERSION"
+  # RHEL 8 uses python3.
+  elif [ "$RPM_DIST_NAME" = "rhel" -a "$RPM_DIST_VERSION" -ge 8 ]; then
```

Comment:
Following discussions in https://github.com/certbot/certbot/pull/7282#discussion_r308478388, I would like to avoid to repeat two times the same logic about python 3 bootstrapping for RPM distributions, but @bmw would like to avoid also a big and unreadable if condition when this bootstrap is triggered.

So, could you first set a truthy variable `RPM_USE_PYTHON_3` for the each following branches in a if/elif statement:
* fedora 29+
* rhel 8+
* rpm distrib with only python 2.6 (rhel6/centos 6)

Then, if `RPM_USE_PYTHON_3` is truthy, apply the python 3 rpm bootstrap, otherwise apply the python 2 rpm bootstrap ?

Once done, @bmw will update #7282 accordingly.

Reply:
Would a case statement work?  There may be other distros such as SLES 15 that use python 3 as well.

Reply:
I've updated the script with the changes suggested.  I also changed how RPM_DIST_VERSION gets set as I don't think it makes sense to set it to 0 and then immediately change it two lines later.